consul-ribbon/src/main/java/com/smoketurner/dropwizard/consul/ribbon/RibbonJerseyClientBuilder.java

```diff
@@ -15,68 +15,71 @@
  */
 package com.smoketurner.dropwizard.consul.ribbon;
 
-import java.util.Objects;
-import javax.annotation.Nonnull;
-import javax.ws.rs.client.Client;
 import com.netflix.loadbalancer.Server;
 import com.netflix.loadbalancer.ZoneAwareLoadBalancer;
 import com.orbitz.consul.Consul;
 import io.dropwizard.client.JerseyClientBuilder;
 import io.dropwizard.lifecycle.Managed;
 import io.dropwizard.setup.Environment;
 
+import javax.annotation.Nonnull;
+import javax.ws.rs.client.Client;
+import java.util.Objects;
+
 public class RibbonJerseyClientBuilder {
 
     private final Environment environment;
     private final Consul consul;
+    private final RibbonLoadBalancerConfiguration configuration;
 
     /**
      * Constructor
-     *
-     * @param environment
+     *  @param environment
      *            Dropwizard environment
      * @param consul
-     *            Consul client
+     * @param configuration
      */
     public RibbonJerseyClientBuilder(@Nonnull final Environment environment,
-            @Nonnull final Consul consul) {
+                                     @Nonnull final Consul consul,
+                                     @Nonnull final RibbonLoadBalancerConfiguration configuration) {
         this.environment = Objects.requireNonNull(environment);
         this.consul = Objects.requireNonNull(consul);
+        this.configuration = Objects.requireNonNull(configuration);
     }
 
     /**
      * Builds a new {@link RibbonJerseyClient}
      *
-     * @param configuration
-     *            Ribbon Load Balancer configuration
+     * @param clientName Jersey client name
+     * @param serviceDiscoverer
+     *            Service discoverer
      * @return new RibbonJerseyClient
      */
-    public RibbonJerseyClient build(
-            @Nonnull final RibbonLoadBalancerConfiguration configuration) {
+    public RibbonJerseyClient build(@Nonnull final String clientName,
```

Comment:
maybe we can provide another `build()` signature that provides a default implementation for `ConsulServiceDiscoverer` that returns the healthy instances since that would be a common use case.  What do you think?